### PR TITLE
Pep8 and type cleanup in instruments

### PR DESCRIPTION
**Pull request by jenshnielsen**
Was looking over this code and noticed a few issues. 

There are still more things that could be improved but this is imho a clean improvement

### CODE REVIEW

File: qcodes/instrument/channel.py [hunk 1]
```diff
@@ -787,30 +789,31 @@ class AutoLoadableChannelList(ChannelList):
...
+        ValueError: If :class:`chan_type` is not a subclass of
+            :class:`InstrumentChannel`
         ValueError: If ``multichan_paramclass`` is not a subclass of
-            :class:`MultiChannelInstrumentParameter` (note that a class is a subclass
-            of itself).
-
+            :class:`MultiChannelInstrumentParameter` (note that a class is a
+            subclass of itself).
     """
     def __init__(
```
Comment (jenshnielsen) [inline qcodes/instrument/channel.py]: I am not a big fan of this indention style but left it for now

File: qcodes/instrument/channel.py [hunk 2]
```diff
@@ -732,33 +734,33 @@ def remove(self)->None:
...
         raise NotImplementedError("Please subclass")
 
-    def _assert_existence(self)->None:
+    def _assert_existence(self) -> None:
         if not self._exists_on_instrument:
             raise RuntimeError(
                 "Object does not exist (anymore) on the instrument")
 
-    def write(self, cmd: str)->Any:
+    def write(self, cmd: str):
```
Comment (WilliamHPNielsen) [inline qcodes/instrument/channel.py]: Why not ` -> None` for this one? Is `mypy` unhappy about the `super()`?

Comment (jenshnielsen) [reply qcodes/instrument/channel.py]: Because @astafan8 is going to tell me that its redundant :)

File: qcodes/instrument/channel.py (hunk 1 above)
Comment (astafan8) [reply qcodes/instrument/channel.py]: what would you recommend for the case of many arguments each having long type annotations?

Comment (jenshnielsen) [reply qcodes/instrument/channel.py]: One per line is fine but this ( on separate lines thing is not great. These lines are actually already too long even with this style

**Pull request merged**